sdk/examples/wallet/offline_signing/3_send_transaction.rs

```diff
@@ -3,57 +3,54 @@
 
 //! In this example we send the signed transaction in a block.
 //!
-//! `cargo run --example 3_send_transaction --release`.
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --all-features --example 3_send_transaction
+//! ```
 
-use std::{fs::File, io::prelude::*, path::Path};
+use std::env::var;
 
 use iota_sdk::{
     client::{
         api::{SignedTransactionData, SignedTransactionDataDto},
         Client,
     },
-    wallet::{Result, Wallet},
+    types::block::payload::transaction::TransactionId,
+    wallet::{Account, Result, Wallet},
+};
+use tokio::{
+    fs::File,
+    io::{AsyncReadExt, BufReader},
 };
 
-const SIGNED_TRANSACTION_FILE_NAME: &str = "examples/wallet/offline_signing/signed_transaction.json";
+// TODO: .env file?
```

Comment:
Decided against putting those into `.env` files because despite putting those paths into consts at the top it's not actually something that makes a lot of sense for the user to change.